src/main/java/frc/utils/AutoSelector.Java

```diff
@@ -0,0 +1,470 @@
+        }
+    }
+    public boolean isAtRocketShip() {
+        if(OI.dial10.get()) {
+            return true;
+        } else { 
+            return false;
+        }
+    }
+    public boolean isRocketRocket() {
+        if(OI.dial11.get()) {
+            return true;
+        } else { 
+            return false;
+        }
+    }
+
+    public void selectAuto() {
+    //High left start L1
+    //LEVEL 1!!
+    //LEVEL 1!!
+    //LEVEL 1!!
+    //CARGO!!!!!!!!!!!!
+    if (ifLoadCargo && isAtHighLeft() && isRocketRocket()) {
+        autoCommand = new LStart_LRocket_LRocket();
+    }
+    
+    if (ifLoadCargo && isAtHighLeft() && isAtRocketShip()) {
+        autoCommand = new LStart_LRocket_LShip();
+    }
+   
+    if (ifLoadCargo && isAtHighLeft() && isAtShipRocket()) {
+        autoCommand = new LStart_LShip_LRocket();
+    }
+    
+    if (ifLoadCargo && isAtHighLeft() && isAtShipShip()) {
+        autoCommand = new LStart_LShip_LShip();
+    }
+
+    //Low Left Start L1
+    if (ifLoadCargo && isAtLowLeft() && isRocketRocket()) {
+        autoCommand = new LStart_LRocket_LRocket();
+    }
+
+    if (ifLoadCargo && isAtLowLeft() && isAtRocketShip()) {
+        autoCommand = new LStart_LRocket_LShip();
+    }
+    
+    if (ifLoadCargo && isAtLowLeft() && isAtShipRocket()) {
+        autoCommand = new LStart_LShip_LRocket();
+    }
+
+    if (ifLoadCargo && isAtLowLeft() && isAtShipShip()) {
+        autoCommand = new LStart_LShip_LShip();
+    }
+
+    //Center Start L1
+    if (ifLoadCargo && isAtCenter() && isRocketRocket()) {
+        autoCommand = new LStart_LRocket_LRocket();
+    }
+
+    if (ifLoadCargo && isAtCenter() && isAtRocketShip()) {
+        autoCommand = new LStart_LRocket_LShip();
+    }
+
+    if (ifLoadCargo && isAtCenter() && isAtShipRocket()) {
+        autoCommand = new LStart_LShip_LRocket();
+    }
+
+    if (ifLoadCargo && isAtCenter() && isAtShipShip()) {
+        autoCommand = new LStart_LShip_LShip();
+    }
+
+    //High Right L1
+    if (ifLoadCargo && isAtHighRight() && isRocketRocket()) {
+        autoCommand = new LStart_LRocket_LRocket();
+    }
+
+    if (ifLoadCargo && isAtHighRight() && isAtRocketShip()) {
+        autoCommand = new LStart_LRocket_LShip();
+    }
+
+    if (ifLoadCargo && isAtHighRight() && isAtShipRocket()) {
+        autoCommand = new LStart_LShip_LRocket();
+    }
+
+    if (ifLoadCargo && isAtHighRight() && isAtShipShip()) {
+        autoCommand = new LStart_LShip_LShip();
+    }
+
+    //Low Right L1
+    if (ifLoadCargo && isAtLowRight() && isRocketRocket()) {
+        autoCommand = new LStart_LRocket_LRocket();
+    }
+
+    if (ifLoadCargo && isAtLowRight() && isAtRocketShip()) {
+        autoCommand = new LStart_LRocket_LShip();
+    }
+
+    if (ifLoadCargo && isAtLowRight() && isAtShipRocket()) {
+        autoCommand = new LStart_LShip_LRocket();
+    }
+
+    if (ifLoadCargo && isAtLowRight() && isAtShipShip()) {
+        autoCommand = new LStart_LShip_LShip();
+    }
```

Comment:
are there just not enough auto commands written to fill out the selector? cause you have the same 4 repeating